src/python/grpcio/grpc/_server.py

```diff
@@ -255,6 +256,27 @@ def invocation_metadata(self):
     def peer(self):
         return _common.decode(self._rpc_event.operation_call.peer())
 
+    def peer_identity(self):
+        ids = cygrpc.peer_identity(self._rpc_event.operation_call)
+        if not ids:
```

Comment:
`return ids if ids else None`? What not-`None`-but-false-in-a-boolean-context value is being screened out here anyway?